Lib/test/libregrtest/save_env.py

```diff
@@ -65,8 +66,40 @@ def __init__(self, testname, verbose=0, quiet=False, *, pgo=False):
                  'sysconfig._CONFIG_VARS', 'sysconfig._INSTALL_SCHEMES',
                  'files', 'locale', 'warnings.showwarning',
                  'shutil_archive_formats', 'shutil_unpack_formats',
+                 'asyncio.get_event_loop_policy', 'asyncio.get_event_loop',
+                 'asyncio_get_exception_handler', 'asyncio_get_debug',
+                 'asyncio_get_child_watcher',
                 )
 
+    def get_asyncio_get_event_loop_policy(self):
+        return asyncio.get_event_loop_policy()
```

Comment:
This function creates a new policy if there is no current policy. The function has a side effect, so I don't think that it's ok to call it from here.